Website.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask
+import random
+
+
+app = Flask(__name__)
+random.seed(0)
+words = ["red","blue","green","brown"]
```

Comment:
typical style has spaces after commas in arrays, e.g.,
`["red", "blue", "green", "brown"]`